src/server.ts

```diff
@@ -358,12 +366,26 @@ export class Server {
 
       let plugin: Plugin;
       let constructor: AccessoryPluginConstructor;
+
       try {
         plugin = this.pluginManager.getPluginForAccessory(accessoryIdentifier);
+      } catch (error) {
+        log.error(`No plugin was found for the accessory "${accessoryIdentifier}" in your config.json at position ${index + 1}. Please make sure the plugin is installed correctly.`);
```

Comment:
Adjusted code so it will handle these error messages properly. Also made changes so it can handle the case where one of the disabled plugins might share an identifier with an enabled plugin/

Reply:
Great 👍